app/models/approval_request.rb

```diff
@@ -4,6 +4,10 @@ class ApprovalRequest < ApplicationRecord
 
   belongs_to :order_item
 
+  after_save do
```

Comment:
@gmcculloug Since the order_item is updated many times would each save end up logging this message.

Reply:
Good catch, it should be `after_create`.  Thanks.